plugins/woocommerce/client/blocks/tests/e2e/plugins/additional-checkout-fields.php

```diff
@@ -103,12 +103,15 @@ public function register_custom_checkout_fields(): void {
 				'location' => 'order',
 				'type'     => 'checkbox',
 				'hidden'   => array(
-					'cart' => array(
-						'properties' => array(
-							'totals' => array(
-								'properties' => array(
-									'totalPrice' => array(
-										'maximum' => 4000,
+					'type'       => 'object',
+					'properties' => array(
+						'cart' => array(
+							'properties' => array(
+								'totals' => array(
+									'properties' => array(
+										'totalPrice' => array(
+											'maximum' => 4000,
+										),
```

Comment:
I am wondering whether it would be beneficial for tests to cover both wrapped and unwrapped objects?

Reply:
I'm in favour of this since its in an area of extensibility so we'd want that safety net for backcompat. Wonder if this test case would be better suited for a unit/integration test though.

Reply:
I agree, I will expand the tests.